### ISSUE TITLE
Enable keyboard control in libraries.

### ISSUE THREAD

**Issue by quachtina96** [LLK/scratch-gui#907]
### Expected Behavior
You should be able to 
- tab from the search input to the close button
- move through the library items via tabbing 
- select a focused library item by pressing 'space' or 'enter'
- move spatially (up, down, left, right) through the library using arrow keys. 

### Actual Behavior
cannot tab beyond search input

### Steps to Reproduce
Open up Costume, Sound, or Sprite Library and press tab.

### Operating System and Browser
Mac OS 10.12.6 Chrome 62.0.3202.89

**Comment by paulkaplan** [comment]
I'm going to close this since you can tab through the library items, but I'll leave https://github.com/LLK/scratch-gui/issues/1411 open to cover the request for keyboarding around the libraries.
